js/modules/loki_app_dot_net_api.js

```diff
@@ -254,6 +266,14 @@ class LokiAppDotNetServerAPI {
     return res.response.data.annotations || [];
   }
 
+  async setAvatar(url, profileKey) {
+    let value = null;
```

Comment:
This needs to be undefined. A null is preserved but an undefined will clean up the complete annotation. This is post Sacha, I can change this after the PR.